### DISCUSSION TITLE
Why in design-system example node-linked in .npmrc is set to hoisted?

### DISCUSSION THREAD

**Discussion by its-monotype** [vercel/turborepo#5307] [Help] [answered]
https://github.com/vercel/turbo/blob/d859a8b26663fc8231c41a8dcebcb7f4eee2238d/examples/design-system/.npmrc#LL2C11-L2C11

Is there any reason for this?

**Comment by mehulkar** [comment, marked as answer]
This has been removed, I'm not sure there was a good reason for this before! cc @tknickman just in case he knows

Changed here: https://github.com/vercel/turbo/pull/5884/files
